include/STREAM/Equations/ConfinementTime.hpp

```diff
@@ -25,15 +24,51 @@ namespace STREAM{
                         EllipticalRadialGridGenerator *radials;
 						DREAM::IonHandler *ions;
                         
-                        len_t id_Tcold, id_Wi, id_Ni; 
+                        len_t id_Tcold, id_Wi, id_Ni, id_Ip, id_Ncold, id_Efield; 
                         
                         ConnectionLength *CL;
                         
                         len_t D_index;
+                        
+                        static constexpr real_t ConversionFactor = 1e-6;
+                        static constexpr real_t nullset = 0.0;
+                        static constexpr real_t Conv_n20 = 1e-20;
+                        
+                        bool mixte;
```

Comment:
should probably stick with variable names in English also with more descriptive names:

mixed_conf_law 

for instance.

Reply:
It has been changed

Reply:
wondering if we should instead call this 'max_perp_tau' or something. We are not mixing two laws, we are taking the maximum of two perpendicualr energy confinement times, no?

Reply:
great. Thanks Felicien this has been changed to maxPerpLaw.